website_sale_secondary_unit/security/ir.model.access.csv

```diff
@@ -1,5 +1,5 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_product_secondary_unit_public,access_product_second_unit_public,product_secondary_unit.model_product_secondary_unit,base.group_public,1,0,0,0
 access_product_secondary_unit_portal,access_product_second_unit_portal,product_secondary_unit.model_product_secondary_unit,base.group_portal,1,0,0,0
-access_product_uom_public,access_product_uom_public,product.model_product_uom,base.group_public,1,0,0,0
-access_product_uom_portal,access_product_uom_portal,product.model_product_uom,base.group_portal,1,0,0,0
+access_product_uom_public,access_product_uom_public,uom.model_uom_uom,base.group_public,1,0,0,0
```

Comment:
Can we avoid to give this permission?

Reply:
If I remove this permission I have to do all uom computations with sudo...

Reply:
But if they are done on the controller, I think it's good enough, isn't it? The rule is to avoid giving open permissions to portal/public users. I know it's not a critical model to be read, but...

Reply:
This question still remains

Reply:
You can try it on runbot